### PR TITLE
DMP-717 : Fix(css) - change css on homepage table

### PR DESCRIPTION
**Pull request by bastien-landre**

### CODE REVIEW

File: src/_table.scss [hunk 1]
```diff
@@ -28,13 +28,11 @@ table.list {
       &.multiple-odd {
         > td:first-child {
           padding-left: 20px;
-          border-left: 10px solid $brand-primary;
```
Comment (adlenafane) [inline src/_table.scss]: Do we really want it?

Comment (adlenafane) [inline src/_table.scss]: It's used on the retailer pish list

Comment (birdy-) [inline src/_table.scss]: Yes @tita validated it

**Comment by birdy-** [comment]
LGTM

**Comment by tita** [comment]
Hi!

Can you delete my mail address? I think you have a wrong entry in your
distributor!

Yours Christoph

Von meinem iPhone gesendet

Am 28.09.2015 um 12:17 schrieb Adlen Afane <notifications@github.com>:

Merged #75 <https://github.com/alkemics/alk-style-common/pull/75>.

—
Reply to this email directly or view it on GitHub
<https://github.com/alkemics/alk-style-common/pull/75#event-420586215>.

**Pull request merged**